tck/src/main/java/org/eclipse/sparkplug/tck/test/host/SessionTerminationTest.java

```diff
@@ -174,7 +175,7 @@ public void disconnect(final @NotNull String clientId, final @NotNull Disconnect
 
 		if (hostClientId == null) {
 			logger.warn("{} host application clientid is not set", getName());
-			return;
+			theTCK.endTest();
```

Comment:
I don't think we want to do that, do we? If we get any disconnect from any client at all, the test will end. Won't work well with other MQTT traffic.

Reply:
Yes - makes sense - fixing.